cmd/vcfFilter/vcfFilter.go

```diff
@@ -212,6 +214,20 @@ func getTests(c criteria, header vcf.Header) testingFuncs {
 			})
 	}
 
+	if c.minDaf != 0 {
+		answer = append(answer,
+			func(v vcf.Vcf) bool {
+				return popgen.VcfSampleDerivedAlleleFrequency(v) > c.minDaf
+			})
+	}
+
+	if c.maxDaf != 1 {
+		answer = append(answer,
+			func(v vcf.Vcf) bool {
+				return popgen.VcfSampleDerivedAlleleFrequency(v) < c.maxDaf
+			})
+	}
+
```

Comment:
Is there a check for maxDaf > minDaf, and have an error if false? Would that be necessary, or would you allow maxDaf < minDaf in some cases?

Reply:
fixed in next commit